.gitlab-ci.yml

```diff
@@ -115,189 +105,20 @@ build-gem:
-  extends: .package-arm
-  needs:
-    - build-gem
-    - install-dependencies-arm64
-  script:
-    - ls ../pkg
-    - cp ../lib-injection/host_inject.rb ../tmp
-    - export RUBY_PACKAGE_VERSION=$(cat ../tmp/version.txt)
-    - ../.gitlab/build-deb-rpm.sh
-    - find . -iregex '.*\.\(deb\|rpm\)' -printf '%f\0' | xargs -0 dd-pkg lint
-
-package-oci-amd64:
-  extends: .package-oci
-  stage: package
-  needs:
-    - build-gem
-    - install-dependencies-amd64
-  variables:
-    ARCH: amd64
-  script:
-    - cp ../lib-injection/host_inject.rb ../tmp
-    - export RUBY_PACKAGE_VERSION=$(cat ../tmp/version.txt)
-    - ../.gitlab/build-oci-package.sh
-
-package-oci-arm64:
-  extends: .package-oci
-  stage: package
+package-oci:
   needs:
     - build-gem
-    - install-dependencies-arm64
-  variables:
-    ARCH: arm64
-  script:
-    - cp ../lib-injection/host_inject.rb ../tmp
-    - export RUBY_PACKAGE_VERSION=$(cat ../tmp/version.txt)
-    - ../.gitlab/build-oci-package.sh
-
-oci-internal-publish:
-  extends: .oci-internal-publish
-  stage: package
-  needs: [ package-oci-arm64, package-oci-amd64 ]
-  rules:
-    - when: on_success
-  variables:
-    FLAVOR: datadog-apm-library-ruby
-
-oci-internal-test-ecr-publish:
-  stage: package
-  needs: [ oci-internal-publish ]
-  rules:
-    - when: on_success
-  trigger:
-    project: DataDog/public-images
-    branch: main
-    strategy: depend
-  variables:
-    IMG_SOURCES: registry.ddbuild.io/ci/remote-updates/datadog-apm-library-ruby:pipeline-${CI_PIPELINE_ID}-1
-    IMG_DESTINATIONS: apm-library-ruby-package:pipeline-${CI_PIPELINE_ID}
-    IMG_REGISTRIES: agent-qa
+    - install-dependencies
 
-onboarding_tests:
-  extends: .base_job_onboarding
-  stage: integration-tests
-  needs: [ oci-internal-test-ecr-publish ]
-  allow_failure: false
-  variables:
-    TEST_LIBRARY: ruby
-    SCENARIO: SIMPLE_INSTALLER_AUTO_INJECTION
-    DD_INSTALLER_LIBRARY_VERSION: pipeline-${CI_PIPELINE_ID}
+onboarding_tests_installer:
   parallel:
-      matrix:
-        - ONBOARDING_FILTER_WEBLOG: [test-app-ruby, test-app-ruby-container]
-  script:
-    - git clone https://git@github.com/DataDog/system-tests.git system-tests
-    - cd system-tests
-    - ./build.sh -i runner
-    - timeout 2700s ./run.sh $SCENARIO --vm-weblog ${ONBOARDING_FILTER_WEBLOG} --vm-env prod --vm-library ${TEST_LIBRARY} --vm-provider aws --vm-skip-branches ubuntu18_amd64
-
-.release-package:
-  stage: deploy
-  variables:
-    PRODUCT_NAME: auto_inject-ruby
-
-save_versions:
-  image: $DOCKER_REGISTRY/images/mirror/ruby:3.2.2
-  stage: deploy
-  needs: []
-  rules:
-    - if: $CI_PIPELINE_SOURCE == "schedule"
-      when: on_success
-    - when: manual
-      allow_failure: true
-  script:
-    - git fetch --all
-    - echo COMMIT_SHA_1X=$(git rev-parse origin/1.x-stable) >> upstream.env
-    - echo COMMIT_SHA_2X=$(git rev-parse origin/master) >> upstream.env
-  artifacts:
-    paths:
-      - upstream.env
-
-deploy_to_reliability_env:
```

Comment:
`deploy_to_reliability_env` has been moved to the shared pipeline as well. 

However, in order to support multi branch deployment, we are getting the commit shas from branches in `save_versions` job. 

How can we provide this information to the `deploy_to_reliability_env` job from the  shared pipeline?

Reply:
implemented